server/controllers/index.js

```diff
@@ -18,5 +19,6 @@ router.get('/check-auth', auth, checkAuth)
 router.post('/api/login', login.get)
 router.post('/api/pharmacy/medicine', auth, addMedicine)
 router.patch('/api/pharmacy/medicine/:medicineid', auth, updateMedForm)
+router.get('/api/clearcookies', clearCookies.get)
```

Comment:
I would use logout as a name for the handler instead of clearCookies, same for the endpoint